nestjs/src/courses/interviews/interviews.controller.ts

```diff
@@ -96,4 +106,59 @@ export class InterviewsController {
 
     throw new BadRequestException('Invalid interview id');
   }
+
+  // use `type` as a way to differentiate between stage-interview and interview.
+  @Get('/:interviewId/:type/feedback')
+  @ApiOkResponse({ type: InterviewFeedbackDto })
+  @ApiForbiddenResponse()
+  @ApiBadRequestResponse()
+  @ApiOperation({ operationId: 'getInterviewFeedback' })
+  @RequiredRoles([CourseRole.Mentor, CourseRole.Supervisor, CourseRole.Manager])
```

Comment:
@RequiredRoles([CourseRole.Mentor, CourseRole.Supervisor, CourseRole.Manager], true)

Reply:
you are my double hero, thanks!